Pipfile

```diff
@@ -4,9 +4,9 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-CoreDataModules = {editable = true,git = "https://www.github.com/AfricasVoices/CoreDataModules",ref = "v0.16.0",extras=["mapping"]}
+CoreDataModules = {editable = true,git = "https://www.github.com/AfricasVoices/CoreDataModules",ref = "enable-copying-code-schemes",extras=["mapping"]}
 RapidProTools = {editable = true,git = "https://www.github.com/AfricasVoices/RapidProTools",ref = "v0.3.9"}
-CodaV2PythonClient = {editable = true,git = "https://www.github.com/AfricasVoices/CodaV2PythonClient",ref = "enable-set-segment-count-to-run-in-transaction"}
+CodaV2PythonClient = {editable = true,git = "https://www.github.com/AfricasVoices/CodaV2PythonClient",ref = "main"}
```

Comment:
How far are we from being able to release these dependency updates

Reply:
I've released the CodaV2PythonClient updates